assets/css/free-trial-admin.css

```diff
@@ -0,0 +1,13 @@
+.wc-calypso-notice {
```

Comment:
I could not use `notice` class from WordPress core. Wocommerce Admin hides core notices selectively according to this [PR](https://github.com/woocommerce/woocommerce-admin/issues/4005). When I add `notice` class, the div gets hijacked to a different parent.